src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -110,72 +152,136 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
+                        "method": "val"
+                    }
+                }
             }
         },
         listeners: {
-            // This can be removed once the jQuery UI slider has built in ARIA
-            "onCreate.initSliderAria": {
-                "this": "{that}.dom.thumb",
+            "onCreate.initSliderAttributes": {
+                "this": "{that}.container",
                 method: "attr",
                 args: [{
-                    role: "slider",
-                    "aria-valuenow": "{that}.combinedSliderOptions.value",
-                    "aria-valuemin": "{that}.combinedSliderOptions.min",
-                    "aria-valuemax": "{that}.combinedSliderOptions.max"
+                    "value": "{that}.model.value",
+                    "min": "{that}.combinedSliderOptions.min",
+                    "max": "{that}.combinedSliderOptions.max",
+                    "step": "{that}.combinedSliderOptions.step",
+                    "type": "range"
                 }]
             },
             "onCreate.bindSlideEvt": {
-                "this": "{that}.slider",
-                "method": "bind",
-                "args": ["slide", "{that}.setModel"]
+                "this": "{that}.container",
+                "method": "on",
+                "args": ["input", "{that}.setModel"]
+            },
+            "onCreate.bindRangeChangeEvt": {
+                "this": "{that}.container",
+                "method": "on",
+                "args": ["change", "{that}.setModel"]
             }
         },
         modelListeners: {
             "value": [{
-                listener: "{that}.setSliderValue",
-                args: ["{change}.value"]
-            }, {
-                listener: "{that}.setSliderAria",
-                args: ["{change}.value"]
+                "this": "{that}.container",
+                "method": "val",
+                args: ["{change}.value"],
+                // If we don't exclude init, the value can get
+                // set before onCreate.initSliderAttributes
+                // sets min / max / step, which messes up the
+                // initial slider rendering
+                excludeSource: "init"
             }]
         }
     });
 
+    fluid.defaults("fluid.slider.jQuery", {
+            gradeNames: ["fluid.slider"],
+            selectors: {
+                thumb: ".ui-slider-handle"
+            },
+            members: {
+                slider: {
+                    expander: {
+                        "this": "{that}.container",
+                        method: "slider",
+                        args: ["{that}.combinedSliderOptions"]
+                    }
+                }
+            },
+            invokers: {
+                setSliderValue: {
+                    "this": "{that}.slider",
+                    "method": "slider",
+                    args: ["value", "{arguments}.0"]
+                },
+                setSliderAria: {
```

Comment:
Maybe this should be renamed to indicated that it is only setting the "aria-valuenow" attribute. It is a bit confusing as is, because of all the other aria attributes that are set onCreate.